benchmark/wildtime_benchmarks/example_pipelines/arxiv.yaml

```diff
@@ -41,7 +41,7 @@ data:
   dataset_id: arxiv
   bytes_parser_function: |
     def bytes_parser_function(data: bytes) -> str:
-      return data.decode("utf-8")
+      return str(data, "utf8")
```

Comment:
Any particular reason to change this, in my experience `bytes.decode(...)` is more idiomatic?https://stackoverflow.com/a/14473492

Reply:
hmm I should change the type hint...
the reason is that `data.decode` does not work here as the input is not bytes but a `memoryView` object that has no `.decode` method...